spec/contracts/notifications/create_contract_spec.rb

```diff
@@ -73,46 +67,17 @@
     end
 
     context 'without a reason for IAN with read_ian false' do
```

Comment:
The context description needs to be updated. 

`read_ian` does not factor in into the `reason` calculation any more.